### PR TITLE
Scheduled Reminders

### PR DESCRIPTION
**Pull request by kraftp**

### CODE REVIEW

File: python/scheduled-reminders/scheduled_reminders/main.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+@DBOS.step()
+def send_email(to_email: str, time: str):
+    message = Mail(
+        from_email=from_email,
+        to_emails=to_email,
+        subject="DBOS Reminder",
+        html_content=f"This is a reminder from DBOS! It has been {time} since your last reminder.",
+    )
+    email_client = SendGridAPIClient(api_key)
```
Comment (maxdml) [inline python/scheduled-reminders/scheduled_reminders/main.py:72]: Why not use a single, long live client?

File: python/scheduled-reminders/start_postgres_docker.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            "run",
+            "--rm",
+            "--name=dbos-db",
+            f'--env=POSTGRES_PASSWORD={os.environ["PGPASSWORD"]}',
+            "--env=PGDATA=/var/lib/postgresql/data",
+            "--volume=/var/lib/postgresql/data",
+            "-p",
+            f"{port}:5432",
+            "-d",
+            "pgvector/pgvector:pg16",
```
Comment (maxdml) [inline python/scheduled-reminders/start_postgres_docker.py:32]: No need pgvector

**Review by maxdml — APPROVED** [review]

File: python/scheduled-reminders/scheduled_reminders/main.py (hunk 1 above)
Comment (kraftp) [reply python/scheduled-reminders/scheduled_reminders/main.py:72]: This is how their docs recommended

File: python/scheduled-reminders/start_postgres_docker.py (hunk 2 above)
Comment (kraftp) [reply python/scheduled-reminders/start_postgres_docker.py:32]: We're using the same script everywhere

**Review by qianl15 — APPROVED** [review]

**Pull request merged**